stats/views.py

```diff
@@ -113,18 +113,14 @@ def get_context_data(self, **kwargs):
         tenses = scenario.mds_labels
         fragment_pks = scenario.mds_fragments
 
-        # Retrieve Fragments, but keep order intact
-        # Solution taken from https://stackoverflow.com/a/38390480
-        preserved = Case(*[When(pk=pk, then=pos) for pos, pk in enumerate(fragment_pks)])
-        fragments = list(Fragment.objects.filter(pk__in=fragment_pks).
-                         order_by(preserved).
-                         prefetch_related('sentence_set', 'sentence_set__word_set'))
+        # TODO: This is not an efficient solution, but it works. Will get back to the previous
```

Comment:
Yes, not sure why this was changed :-)

Reply:
I must have had a reason :man_shrugging: changed it back.